roles/nova-data/tasks/libvirt.yml

```diff
@@ -95,3 +225,12 @@
 
 - name: add nova to the libvirtd group
   user: name=nova groups=libvirtd append=true system=yes createhome=no
+
+- name: ufw allow libvirtd ports for live migration
+  ufw:
+    rule: allow
+    port: "{{ item }}"
+    proto: tcp
+  with_items:
+    - 16509
+    - 16514
```

Comment:
Lets tag this with `ufw`. There was a PR recently that added ufw tags to all these tasks for easy targeting.